_deployment/checklist/012_common_configuration.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Common Configuration Steps
+permalink: "deployment/checklist/commonconfiguration.html"
+layout: "document"
+categories: ["deployment", "checklist", "tds"]
+---
+
+# Common TDS Configuration Steps
+
+The following are instructions for steps for common configuration changes in TDS.
+
+## Changing Assessment Labels in TDS
+Assessment labels are defined in the `configs.client_testproperties` table. The following steps can be taken
+to update the label for a particular assessment id:
+
+1. Update the `label` column in the `config.client_testproperties` 
+```
+UPDATE configs.client_testproperties 
+SET label = '<desired assessment label>' 
+WHERE testid = '<assessment id>'; 
+``` 
+2. Flush the redis cache
```

Comment:
Do we have steps for flushing the redis cache somewhere?  If not, I will add them in a follow up PR

Reply:
We do not. I would add them to this deployment checklist, but I'm not sure if this particular section is the appropriate location for those instructions.